mesonbuild/mintro.py

```diff
@@ -107,9 +107,6 @@ def list_installed(installdata: backends.InstallData) -> T.Dict[str, str]:
         for t in installdata.targets:
             res[os.path.join(installdata.build_dir, t.fname)] = \
                 os.path.join(installdata.prefix, t.outdir, os.path.basename(t.fname))
-            for alias in t.aliases.keys():
-                res[os.path.join(installdata.build_dir, alias)] = \
-                    os.path.join(installdata.prefix, t.outdir, os.path.basename(alias))
```

Comment:
TODO: aliases are now parts of `installdata.symlinks` but those are not part of introspection data.

Reply:
Side effect of this PR, installdata.symlinks are now part of introspection data.